docs/source-app/workflows/byoc/index.rst

```diff
@@ -5,62 +5,270 @@ Run Apps on your own cloud (BYOC)
+  for the custom policy that the custom role uses.
+
+   .. note:: Ignore the warnings generated by AWS when you create the policy.
+
+If you are an IT person creating this role for someone else (researchers, data scientists, Lightning App creators), assign the role to their AWS account after creating the role.
+
+Here's a short video showing you the process.
+
+.. raw:: html
+
+   <video id="background-video" autoplay loop muted controls poster="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/aws_signin.png" width="100%">
+      <source src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/byoc_aws_setup_shorter.mp4" type="video/mp4" width="100%">
+   </video>
+
+----
+
+Method 2: Use Terraform to set up AWS (advanced)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This process is intended for advanced users only. If you have not used Terraform before, you might want to consider using the AWS console.
+
+#. `Download and install Terraform. <https://www.terraform.io/downloads>`_ If you haven’t used Terraform, `here is their tutorial <https://learn.hashicorp.com/collections/terraform/aws-get-started?utm_source=terraform_io_download>`_.
+
+#. Install `AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html>`_.
+
+#. Using the AWS CLI log into your `AWS account <https://aws.amazon.com/free>`_ using `associated credentials <https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html>`_ that allow you to create resources. For example:
+
+    .. code:: bash
+
+        aws configure sso
+        aws sso login
+
+#. Clone the `terraform-aws-lightning-byoc <https://github.com/Lightning-AI/terraform-aws-lightning-byoc>`_ repo.
+
+#. Go to the quick-start directory inside the cloned repo.
+
+#. Run the following commands in the quick-start directory:
+
+    .. code:: bash
+
+        terraform init
+        export AWS_PROFILE=<your profile>
+        aws sts get-caller-identity
+        terraform apply
+
+#. Follow the directions in the CLI.
+
+#. Run the following to get the ARN and external ID:
+
+    .. code:: bash
+
+        terraform out -json
+
+If you are an IT person creating this role for someone else (researchers, data scientists, Lightning App creators), assign the role to their AWS account after creating the role.
+
+----
+
+Get the ARN and external ID
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once IT or you yourself have created the role, you need to get the ARN and external ID from the AWS console.
+
+Here's a short video showing you the process.
+
+.. raw:: html
+
+   <video id="background-video" autoplay loop muted controls poster="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/aws_signin.png" width="100%">
+      <source src="https://pl-flash-data.s3.amazonaws.com/assets_lightning/docs/images/storage/byoc_aws_get_arn_exid.mp4" type="video/mp4" width="100%">
```

Comment:
Again, should we use this S3 bucket for it? How bout storing our videos at the same location where we store other docs assets in the GCP  lightning-docs https://console.cloud.google.com/storage/browser/lightning-docs;tab=objects?forceOnBucketsSortingFiltering=false&project=grid-backend-266721

bucket

Reply:
@nmiculinic This was what I was provided to use. There are actually a lot of videos and images stored in this S3 bucket.